src/kOS/AddOns/KAC/KACWrapper.cs

```diff
@@ -0,0 +1,612 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+
+namespace kOS_KACWrapper
```

Comment:
lets move this into our namespace, that is pretty cool that he built a client library :)